dashboard/public/pixelation/pixelation.js

```diff
@@ -438,6 +445,18 @@ function setSliders() {
   drawGraph();
 }
 
+function setMinTextValues() {
+  if (heightText.value < 1) {
+    heightText.value = 1;
+  }
+  if (widthText.value < 1) {
+    widthText.value = 1;
+  }
+  if (bitsPerPixelText.value < 1) {
+    bitsPerPixelText.value = 1;
+  }
+}
```

Comment:
Any reason that in setSliders we use Math.max, but in setMinTextValues we say if < 1?

I believe heightText.value (and equivalents) is actually a string, that we're then comparing to a number. Can that ever get us into trouble?